backstop/tests/carousel.html

```diff
@@ -0,0 +1,270 @@
+            aria-hidden="false"
+          >
+            <div class="demo-columns">
+              <div class="demo-column demo-left">
+                Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod
+                tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation
+                ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in
+                voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non
+                proident, sunt in culpa qui officia deserunt mollit anim id est laborum.
+              </div>
+              <div class="demo-column demo-right"></div>
+            </div>
+          </li>
+
+          <!-- Slide with full frame video embed. -->
+          <li
+            class="iui-carousel-slider-item demo-carousel-height iui-next"
+            data-slide-id="1"
+            role="group"
+            aria-roledescription="slide"
+            aria-label="2 of 3"
+            aria-hidden="true"
+          >
+            <iframe
+              width="100%"
+              height="100%"
+              src="https://www.youtube.com/embed/DT59xdKNbA8"
+              title="YouTube video player"
+              frameborder="0"
+              allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture"
+              allowfullscreen
+            ></iframe>
+          </li>
+
+          <!-- Slide with image and title. -->
+          <li
+            class="iui-carousel-slider-item demo-carousel-height iui-next"
+            data-slide-id="2"
+            role="group"
+            aria-roledescription="slide"
+            aria-label="3 of 3"
+            aria-hidden="true"
+          >
+            <div
+              class="iui-carousel-slider-item-image"
+              style="background-image: url('../assets/image-test.jpg');"
+            ></div>
+            <div class="iui-carousel-slider-item-title">Bentley System headquarters</div>
+          </li>
+
+        </ol>
+
+        <nav class="iui-carousel-navigation">
+          <button
+            class="iui-button iui-borderless iui-small iui-carousel-navigation-button iui-carousel-navigation-previous"
+            type="button"
+            aria-controls="myCarousel-items"
+            aria-label="Previous slide"
+          >
+            <svg-chevron-left
+              class="iui-button-icon"
+              aria-hidden="true"
+            ></svg-chevron-left>
+          </button>
+
+          <button
+            class="iui-button iui-borderless iui-small"
+            aria-label="Pause animation"
+            data-action="pause"
+          >
+            <svg-pause
+              class="iui-button-icon"
+              aria-hidden="true"
+            ></svg-pause>
+          </button>
+
+          <ol class="iui-carousel-navigation-dots">
+            <li>
```

Comment:
Lets have an example of 4 slides in carousel, just to be sure that everything is ok (as we have 3 classes for slides previous, current and next)

Reply:
Can do.

Reply:
I would even suggest lets have 10 or 20 slides (we could generate them using JS, and even remove the weird Bentley-specific hardcoded content). It would help see the first/second/invisible dots in action too.

Reply:
If we agree to do that, I would definitely need help implementing something like that.

Reply:
Im going to resolve this as now we have more than 3... 
But js is bugged and now it works weirdly, so keeping comment https://github.com/iTwin/iTwinUI/pull/377#discussion_r789997143 open